### ISSUE TITLE
findsecbugs-plugin: missing bug code for keySECXXEVAL

### ISSUE THREAD

**Issue by schloemer-bas** [find-sec-bugs/find-sec-bugs#727]
## Environment

| Component             | Version |
| --------------------- | ------- |
| Maven                 |   3.9.6 |
| Java                  |  11, 17 |
| SpotBugs              |   4.8.3 |
| spotbugs-maven-plugin | 4.8.3.1 |
| findsecbugs-plugin    |  1.13.0 |

## Problem

During a Maven build with goal `spotbugs:spotbugs` in a project that contains a violation of bug SECXXEVAL, the following exception is thrown by spotbugs-maven-plugin:

```
Exception in thread "main" java.lang.IllegalArgumentException: Error: missing bug code for keySECXXEVAL
	at edu.umd.cs.findbugs.DetectorFactoryCollection.getBugCode(DetectorFactoryCollection.java:509)
	at edu.umd.cs.findbugs.SortedBugCollection.writeBugCodes(SortedBugCollection.java:673)
	at edu.umd.cs.findbugs.SortedBugCollection.writeEpilogue(SortedBugCollection.java:579)
	at edu.umd.cs.findbugs.SortedBugCollection.writeXML(SortedBugCollection.java:558)
	at edu.umd.cs.findbugs.SortedBugCollection.writeXML(SortedBugCollection.java:467)
	at edu.umd.cs.findbugs.XMLBugReporter.finish(XMLBugReporter.java:47)
	at edu.umd.cs.findbugs.DelegatingBugReporter.finish(DelegatingBugReporter.java:89)
	at edu.umd.cs.findbugs.DelegatingBugReporter.finish(DelegatingBugReporter.java:89)
	at edu.umd.cs.findbugs.DelegatingBugReporter.finish(DelegatingBugReporter.java:89)
	at edu.umd.cs.findbugs.FindBugs2.analyzeApplication(FindBugs2.java:1165)
	at edu.umd.cs.findbugs.FindBugs2.execute(FindBugs2.java:309)
	at edu.umd.cs.findbugs.FindBugs.runMain(FindBugs.java:395)
	at edu.umd.cs.findbugs.FindBugs2.main(FindBugs2.java:1231)
```

The bug definition for SECXXEVAL was added in #681, and 1.13.0 is the first release after this was merged, so reverting to findsecbugs-plugin 1.12.0 solves the problem.

I am unfamiliar with SpotBugs plugins, but it seems like the problem is that when [plugin/src/main/resources/metadata/messages.xml](https://github.com/find-sec-bugs/find-sec-bugs/pull/681/files#diff-a686f21490e05e3796410ae562a7a8bbc9802a995afee23da568c5d0fe4ff2e4) was changed, a `<BugCode abbrev="SECXXEVAL>` definition should have been added as well.

## Code

Excerpt from `pom.xml`:
```xml
<build>
	<plugins>
	<! ... ->
        <plugin>
            <groupId>com.github.spotbugs</groupId>
            <artifactId>spotbugs-maven-plugin</artifactId>
            <version>4.8.3.1</version>
            <configuration>
                <plugins>
                    <plugin>
                        <groupId>com.h3xstream.findsecbugs</groupId>
                        <artifactId>findsecbugs-plugin</artifactId>
                        <version>1.13.0</version>
                    </plugin>
                </plugins>
            </configuration>
        </plugin>
	<! ... ->
    </plugins>
</build>
```

I do not know exactly which code triggers the bug, so I cannot provide an example for the Java code.

**Comment by pavelorehov** [comment]
Have same issue, will there be a new release soon ?

**Comment by uwesinha** [comment]
Could we maybe get a 1.13.1 release with fix #728 applied and nothing else? This is a small, but very annoying bug that keeps us from upgrading to 1.13.

**Comment by guymahieu** [comment]
This issue is blocking us in our upgrade to Java 21 (due to a false positive), any update on when we could expect a (small?) release with this fix?
